packages/prettier-plugin-sql/src/index.ts

```diff
@@ -22,8 +23,8 @@ export const printers = {
   sql: {
     print(path: AstPath) {
       return (
-        format(path.node, {
-          language: 'postgresql',
+        formatDialect(path.node, {
+          dialect: postgresql,
```

Comment:
Were these changes from Nathan in the other repo? It's a bit suspicious seeing changes to SQL formatters in this LLM PR but I'm ok with it if it's from something that Nathan did.

Reply:
It's something I did; our SQL formatter doesn't support pgvector operators. This allows the format checks to not reject the `<=>` (cosine-distance) operator

Reply:
Oh right, I remember now, thanks for clarifying!